data/filter/wg21.py

```diff
@@ -215,6 +229,10 @@ def build_code(elem, format):
     def build_row(elems):
         return pf.TableRow(*[pf.TableCell(elem) for elem in elems])
 
+    def build_row_content(headers, examples):
```

Comment:
The current state was due to a potential use case of a "header" row in the middle of a table, but it doesn't seem like a good nor common use case.

Do you think we can just turn the headers into proper header row instead? From my testing, it seems like changing the "header row" to a proper header row prevents the splitting over pages.

Reply:
Oh... I see in your example that you actually have a "header" row in the middle of a table 😂 Would you be opposed to simply splitting those into two tables?

Reply:
I was doing separate tables originally. The separate tables used different widths that looked ugly.
could try nested tables where each cell has a table with an optional header and a cell with the code.

Reply:
made real header row instead.